src/components/GalaxyModal.js

```diff
@@ -115,7 +114,7 @@ export const GalaxyModalBase = withDisplayName('GalaxyModal')(
             ['RUNNING', () => h(Fragment, [
               deleteButton,
               pauseButton,
-              h(ButtonPrimary, { disabled: false, onClick: () => setViewMode('launchWarn') }, ['Launch Galaxy'])
+              h(ButtonPrimary, { disabled: false, onClick: () => setViewMode('launchWarn') }, ['Open Galaxy'])
```

Comment:
Should the default text be updated for consistency?

![image](https://user-images.githubusercontent.com/484484/160703837-ece26620-1471-46be-9c30-59eece09238e.png)

Reply:
This seems to be line 391 in `CloudEnvironmentModal`. There are also some tooltips/aria text things in that file that should be changed from Launch -> Open.

Reply:
Here is some more inconsistency.

![image](https://user-images.githubusercontent.com/484484/160705832-53abe60e-51fe-4d00-8e63-c72946ff72da.png)

Reply:
Huh... that occurrence just doesn't show up with ctrl+f for me... interesting. Good catch :). I updated the `Cloud Environment Details` text as well to be context-specific.